scripts/zones/Carpenters_Landing/mobs/Para.lua

```diff
@@ -0,0 +1,97 @@
+-----------------------------------
+--  Area: Carpenters' Landing
+--    NM: Para
+-- Quest: Elderly Pursuits
+-----------------------------------
+local ID = zones[xi.zone.CARPENTERS_LANDING]
+-----------------------------------
+---@type TMobEntity
+local entity = {}
+
+entity.onMobSpawn = function(mob)
+    mob:setMobMod(xi.mobMod.IDLE_DESPAWN, 180) -- 3 minutes
+
+    local para = GetMobByID(ID.mob.PARA)
+
+    if
+        Para:getLocalVar('spawnAdds') < 5
+    then
+        Para:setLocalVar('spawnAdds', Para:getLocalVar('spawnAdds') + 1)
+    end
+
+end
+
+entity.onMobFight = function(mob, target)
+end
+
+--If Para uses TP specific TP moves move under 50% HP then it will spawn a clone.
+--Will only spawn 4 adds total.
+--Adds will spawn ontop of the player.
+--TP moves that spawned a clone in the capture: 310 - Queezyshroom, 311- Numbshroom, 312 - Shakeshroom
+--No reports or captures if the clones can respawn.
+--Reports from players saying only the original needs to be killed.
+entity.onMobWeaponSkill = function(target, mob, skill, player)
```

Comment:
I don't know if CI is set up to catch this, but these comments need spaces. Good comments though 👍

Reply:
Done.